### PR TITLE
Add cloudfactory information

### PR DESCRIPTION
**Pull request by subashshrestha10**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

## Why was this change needed?
I have added Cloudfactory information.

### CODE REVIEW

File: data/cloudfactory.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "CloudFactory",
+  "establishedAt": 1315785600,
+  "location": "Jhamsikhel, Lalitpur, Nepal",
+  "website": "https://www.cloudfactory.com/",
+  "email": "hello@cloudfactory.com",
+  "phone": "01-5592701",
```
Comment (thenishantsapkota) [inline data/cloudfactory.json:7]:
```json
  "phone": "+977-15592701",
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/cloudfactory.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "CloudFactory",
+  "establishedAt": 1315785600,
+  "location": "Jhamsikhel, Lalitpur, Nepal",
```
Comment (thenishantsapkota) [inline data/cloudfactory.json:4]:
```suggestion
  "location": "Jhamsikhel, Lalitpur",
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/cloudfactory.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "CloudFactory",
```
Comment (thenishantsapkota) [inline data/cloudfactory.json:2]:
```suggestion
  "name": "CloudFactory Ltd.",
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/cloudfactory.json (hunk 3 above)
Comment (subashshrestha10) [reply data/cloudfactory.json]: Update the changes

File: data/cloudfactory.json (hunk 2 above)
Comment (subashshrestha10) [reply data/cloudfactory.json]: Update the changes

File: data/cloudfactory.json (hunk 1 above)
Comment (subashshrestha10) [reply data/cloudfactory.json]: Update the changes

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contributions @subashshrestha10  :partying_face:
Join our [Discord](http://discord.gg/7jwZaa8WDr) to claim your contributors role (_if you haven't already_) and stay connected with us.
Cheers!

**Pull request merged**